### ISSUE TITLE
Clarification of "Model used in filtering"

### ISSUE THREAD

**Issue by fajri91** [FomalhautB/KM-BART-ACL#4]
Hi guys, 

Is this the resulting model of the pretraining step (i.e. prior to fine-tuning)?

**Comment by procedure2012** [comment]
Yes, the weights are from full models, which are pretrained with all pretraining tasks.
